### PR TITLE
Init module

### PR DESCRIPTION
**Pull request by jbarascut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+```hcl
+module "grafana" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-grafana.git"
+  cluster_name     = module.eks.cluster_name
+  argocd_namespace = local.argocd_namespace
+  base_domain      = module.eks.base_domain
+  grafana = {
+    oidc = module.oidc.oidc
+  }
+  depends_on = [module.monitoring, module.loki-stack]
```
Comment (ckaenzig) [inline README.md]: Small thing, but correct/clean formatting would be great in examples :wink: 
```suggestion
  source           = "git::https://github.com/camptocamp/devops-stack-module-grafana.git"
  cluster_name     = module.eks.cluster_name
  argocd_namespace = local.argocd_namespace
  base_domain      = module.eks.base_domain
  grafana          = {
    oidc = module.oidc.oidc
  }
  depends_on       = [module.monitoring, module.loki-stack]
```

**Review by ckaenzig — APPROVED** [review]
Nothing important to note, seems good to me.

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  wait = true
+
+  spec {
+    project = argocd_project.this.metadata.0.name
+
+    source {
+      repo_url        = "https://github.com/camptocamp/devops-stack-module-grafana.git"
+      path            = "charts/grafana"
+      target_revision = "init"
```
Comment (lentidas) [inline main.tf:62]: Don´t forget to change this to `target_revision = "main"` before merging!

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+variable "helm_values" {
+  description = "Helm values, passed as a list of HCL structures."
+  type        = any
+  default     = []
+}
+
+variable "dependency_ids" {
+  type = map(string)
+
```
Comment (lentidas) [inline variables.tf:35]: I think we can remove the blank line.

File: main.tf (hunk 2 above)
Comment (lentidas) [inline main.tf]:
```suggestion
      target_revision = "HEAD"
```

**Review by lentidas — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
...
+A [DevOps Stack](https://devops-stack.io) module to deploy and configure [Grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana).
+
+## Usage
+
+```hcl
+module "grafana" {
+  source           = "git::https://github.com/camptocamp/devops-stack-module-grafana.git"
+  cluster_name     = module.eks.cluster_name
+  argocd_namespace = local.argocd_namespace
+  base_domain      = module.eks.base_domain
```
Comment (lentidas) [inline README.md]:
```suggestion
  cluster_name     = var.cluster_name
  argocd_namespace = local.argocd_namespace
  base_domain      = module.cluster.base_domain
```

**Review by lentidas — APPROVED** [review]

**Pull request merged**